tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -20,16 +19,14 @@ public sealed class OrganisationsSteps
     {
         private readonly ScenarioContext _context;
         private readonly Response _response;
-        private readonly IConfigurationRoot _config;
-        private readonly string _organisationUrl = "http://localhost:8075/api/v1/Organisations";
+        private readonly Settings _config;
```

Comment:
Naming of the variable here. Rather than change context could we name it something similar to the class e.g. _settings?